2_natural_language_processing/tests/test_processing.py

```diff
@@ -0,0 +1,34 @@
+"""
```

Comment:
Move this directory under the project directory above, i.e. within `ml_project_similar_words`.